evap/student/views.py

```diff
@@ -310,15 +371,15 @@ def vote(request, evaluation_id):  # noqa: PLR0912
         RatingAnswerCounter.objects.filter(contribution__evaluation=evaluation).update(id=F("id"))
         TextAnswer.objects.filter(contribution__evaluation=evaluation).update(id=F("id"))
 
+        # TODO@Felix: do this in on_commit? (or via select for update?)
```

Comment:
I don't think `on_commit` is needed, the change of `can_publish_text_results` would be rolled back if the transaction fails, right?